test/check-machines-lifecycle

```diff
@@ -349,22 +349,65 @@ class TestMachinesLifecycle(machineslib.VirtualMachinesCase):
         self.waitVmRow("new")
         self.waitVmRow("old", "system", False)
 
-        # Rename to itself and expect error
+        self.goToVmPage("new")
         self.performAction("new", "rename")
 
-        b.set_input_text("#rename-dialog-new-name", "new")
+        # Empty name should be an error
+
+        b.set_input_text("#rename-dialog-new-name", "")
         b.click("#rename-dialog-confirm")
-        b.wait_in_text(".pf-v5-c-modal-box__body .pf-v5-c-alert.pf-m-danger", "Can't rename domain to itself")
+        b.wait_in_text(".pf-v5-c-helper-text__item.pf-m-error", "Name must not be empty")
 
-        self.goToVmPage("new")
-        self.performAction("new", "rename")
+        # Test errors from libvirt
 
-        b.set_input_text("#rename-dialog-new-name", "test%")
+        b.set_input_text("#rename-dialog-new-name", "/")
+        b.click("#rename-dialog-confirm")
+        b.wait_in_text(".pf-v5-c-alert__description", "name / cannot contain '/'")
+
+        # Set full name and description and use non-ASCII characters
+
+        b.set_input_text("#rename-dialog-new-name", 'test%')
+        # This is an attempted injection for the virt-xml invocation
+        b.set_input_text("#rename-dialog-full-name", '"full name",blah=yo')
+        b.set_input_text("#rename-dialog-description", 'Döscrüptiän كرة القدم')
```

Comment:
It would be good to test a multi-line description which also contains a `"` and a `'` to make sure that this goes through virt-xml/xml and back correctly.

Reply:
Yes.

Reply:
Done.